datasets/saudinewsnet/saudinewsnet.py

```diff
@@ -0,0 +1,102 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The dataset contains a set of 2,972 Arabic newspaper articles alongwith metadata, extracted from various online Saudi newspapers."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@misc{hagrima2015,
+author = "M. Alhagri",
+title = "Saudi Newspapers Arabic Corpus (SaudiNewsNet)",
+year = 2015,
+url = "http://github.com/ParallelMazen/SaudiNewsNet"
+}
+"""
+
+
+_DESCRIPTION = """The dataset contains a set of 2,972 Arabic newspaper articles alongwith metadata, \
+extracted from various online Saudi newspapers and written in MSA."""
+
+
+_HOMEPAGE = "https://github.com/parallelfold/SaudiNewsNet"
+
+
+_LICENSE = "Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License."
+
+
+_URL = "https://github.com/parallelfold/SaudiNewsNet/raw/master/dataset/2015-07-21.zip"
+
+
+class SaudiNewsNet(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Saudinewsnet(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of `saudinewsnet`